CHANGELOG.md

```diff
@@ -1,3 +1,104 @@
+- docs: Update README.md ([#5319](https://github.com/reactioncommerce/reaction/pull/5319))
+
+## Chores
+
+- chore: updated Dockerfile to change .meteor/ permissions to the node … ([#5353](https://github.com/reactioncommerce/reaction/pull/5353))
+- chore(ci): add eslint check to only check changed files and fail when warnings are thrown ([#5357](https://github.com/reactioncommerce/reaction/pull/5357))
+- chore(ci): Work more robustly on forks ([#5386](https://github.com/reactioncommerce/reaction/pull/5386))
+- chore: fix no-undef lint issues ([#5360](https://github.com/reactioncommerce/reaction/pull/5360))
+- Update Material-UI 4.x ([#5278](https://github.com/reactioncommerce/reaction/pull/5278))
+- chore: Update README DCO - sign-off not sign ([#5318](https://github.com/reactioncommerce/reaction/pull/5318))
+- chore: update snyk ignore ([#5331](https://github.com/reactioncommerce/reaction/pull/5331))
+- chore: update CHANGELOG, update version ([#5260](https://github.com/reactioncommerce/reaction/pull/5260))
+- chore: use npm command rather than meteor npm ([#5328](https://github.com/reactioncommerce/reaction/pull/5328))
+- chore: update js-doc throughout the app, batch 1 ([#5309](https://github.com/reactioncommerce/reaction/pull/5309))
+- chore: fix all `react/no-deprecated` and `react/prop-type` eslint warnings ([#5307](https://github.com/reactioncommerce/reaction/pull/5307))
+- chore: update reaction-eslint-config version in package.json ([#5320](https://github.com/reactioncommerce/reaction/pull/5320))
+- Organize integration tests ([#5316](https://github.com/reactioncommerce/reaction/pull/5316))
+- chore: add overrides for eslint rules that stlll need fixing ([#5308](https://github.com/reactioncommerce/reaction/pull/5308))
+- chore: fix all `id-length` eslint warnings ([#5298](https://github.com/reactioncommerce/reaction/pull/5298))
+- chore: update lodash to fix snyk critical error ([#5299](https://github.com/reactioncommerce/reaction/pull/5299))
+- chore: fix all `max-len` eslint warnings ([#5296](https://github.com/reactioncommerce/reaction/pull/5296))
+- chore: fix prefer-const eslint error ([#5290](https://github.com/reactioncommerce/reaction/pull/5290))
+
+## Refactors
+
+- refactor: add title as backup in product admin breadcrumb ([#5342](https://github.com/reactioncommerce/reaction/pull/5342))
+- refactor: add isSoldOut prop to variants and options in Catalog ([#5289](https://github.com/reactioncommerce/reaction/pull/5289))
+- refactor: how to get reaction version ([#5324](https://github.com/reactioncommerce/reaction/pull/5324))
+- refactor: fulfillment section of Orders 2.0 ([#5302](https://github.com/reactioncommerce/reaction/pull/5302))
+- refactor: payments section of Orders 2.0 ([#5279](https://github.com/reactioncommerce/reaction/pull/5279))
+
+## Tests
+
+- test: add integration test for ping query ([#5326](https://github.com/reactioncommerce/reaction/pull/5326))
+
+# Breaking Changes
+None
+
```

Comment:
If there were any community contributors to this release, we can note those here.